### ISSUE TITLE
Translate page - enhancement

### ISSUE THREAD

**Issue by mwozny123** [koreader/koreader#2252]
#### Issue
Online translators are getting better and better. 
Why not add option to translate full page with koreader? of course it will require Internet connection.

**Comment by KenMaltby** [comment]
Well we already have Wikipedia, for selected text.  Perhaps, something like that, using Bing or such?  The full page idea might be more of a burden.

**Comment by Frenzie** [comment]
Also note https://www.apertium.org

I have no idea how good it is though.

**Comment by jetboo** [comment]
There is a translator.lua file in /koreader/frontend/ui/ inside which one can read "Translate text using Google Translate" but I don't know what this is used for as we cannot translate any selected word while reading (although this would be GREAT ! )

Is the missing of such feature being related to the Amazon's terms of service ?

**Comment by houqp** [comment]
No, it has nothing to do with amazon's TOS. That part of code is only used for detecting language for a given text, see line 64 in frontend/apps/reader/modules/readerwikipedia.lua. Anyone is welcome to create a google translator plugin using this module.
